numpy/random/generator.pyx

```diff
@@ -3857,12 +3865,17 @@ cdef class Generator:
                     buf[...] = x[j]
                     x[j] = x[i]
                     x[i] = buf
+            x = np.swapaxes(x, 0, axis)
```

Comment:
Because `shuffle` shuffles in-place, this line is pointless - `x` is not used after this point